specification/monitor/resource-manager/Microsoft.Insights/preview/2019-10-17-preview/privateLinkScopes_API.json

```diff
@@ -15,6 +15,9 @@
   "consumes": [
     "application/json"
   ],
+  "x-ms-code-generation-settings": {
```

Comment:
this has been solved and can be removed.

Reply:
The title in readme is `title: MonitorClient`. Any reason why this need to be in json?

Reply:
not sure. But they should have same effect for autorest. We can remove this part since it has been added in the above lines in another PR.

Reply:
if remove this, then in python sdk, the client will be named as: MonitorClient, it will cause a breaking change

Reply:
Then why does this not causing change to other SDKs?

Should we just define another `title: ` in readme inside yaml $(python) block to override the default one?

Reply:
i think it will also cause breaking change to other SDK as Fanqiu mentioned that c# also found this
and i am not sure whether it is reasonable to override it in readme inside $(python), i am not python code-gen, maybe zim is clear about this